src/core/Akka/Actor/CoordinatedShutdown.cs

```diff
@@ -483,26 +481,24 @@ internal static List<string> TopologicalSort(Dictionary<string, Phase> phases)
             var unmarked = new HashSet<string>(phases.Keys.Concat(phases.Values.SelectMany(x => x.DependsOn)));
             var tempMark = new HashSet<string>(); // for detecting cycles
 
-            Action<string> depthFirstSearch = null;
-            depthFirstSearch = u =>
+            void DepthFirstSearch(string u)
```

Comment:
Another local function optimization in `CoordinatedShutdown`. Minimal impact - this code only runs once per application.